tfjs-backend-webgpu/src/base.ts

```diff
@@ -50,7 +50,8 @@ if (isWebGPUSupported()) {
       deviceDescriptor.requiredFeatures = ['timestamp-query'];
     }
     const device: GPUDevice = await adapter.requestDevice(deviceDescriptor);
-    return new WebGPUBackend(device);
+    const adapterInfo = await adapter.requestAdapterInfo();
+    return new WebGPUBackend(device, adapterInfo);
```

Comment:
Question: Can we only pass adapter to WebGPUBackend?

Reply:
We need to allow the user to specify the same device with tfjs to do interop. However, we can't do it if passing adapter to `WebGPUBackend`.